dotnet/src/webdriver/BiDi/Communication/Broker.cs

```diff
@@ -99,13 +101,29 @@ internal Broker(BiDi bidi, ITransport transport)
                 new Json.Converters.Enumerable.GetUserContextsResultConverter(),
                 new Json.Converters.Enumerable.GetClientWindowsResultConverter(),
                 new Json.Converters.Enumerable.GetRealmsResultConverter(),
+            },
+            TypeInfoResolverChain =
```

Comment:
I think `BiDi` can construct his `JsonSerializerContext`, and then `Broker` will rely on it. So I am thinking on a way `completely new own permissions json context` rather than `extending default`. Why: probably having lighter json context is beneficial for serialization in terms of performance.

And Appium (who is most likely on top of Selenium), will also provide their own json context, they are still able to define it on top of our default.

Reply:
Seems good idea. Then We can:
```diff
- internal Broker(BiDi bidi, Uri url)
+ internal Broker(SessionModule session, Uri url)
```

Meaning be closer to `BiDi -> Broker -> Transport`